.github/workflows/coverage.yaml

```diff
@@ -134,3 +134,26 @@ jobs:
           run: |
             FULL_HTML_REPORT_URL=https://canonical.github.io/lxd-ui/coverage
             echo "::notice title=Published Playwright Test Report::$FULL_HTML_REPORT_URL"
+
+
+  tics-report:
+    runs-on: ubuntu-latest
+    needs: publish-coverage-report
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Produce TICS report
+        shell: bash
+        run: |
+          set -x
+          export TICSAUTHTOKEN=${{ secrets.TICS_AUTH_TOKEN }}
+          curl --silent --show-error "https://canonical.tiobe.com/tiobeweb/TICS/api/public/v1/fapi/installtics/Script?cfg=default&platform=linux&url=https://canonical.tiobe.com/tiobeweb/TICS/" > install_tics.sh
+          . ./install_tics.sh
+          TICSQServer -project lxd-ui -tmpdir /tmp/tics -branchdir .
+
+      - name: Upload TICS report
+        if: always()
+        uses: actions/upload-artifact@v4
```

Comment:
This report is huge (55MB), should we set `retension-days` for the `upload-artifact` action? I think 10 days maybe appropriate instead of the default which is 90 days according to our repository setting.

Reply:
Yes, good point. I set it to 7 just now.